File: app/src/main/java/me/kirimin/mitsumine/db/UserIdDAO.java
package me.kirimin.mitsumine.db;

import java.util.ArrayList;
import java.util.List;

import me.kirimin.mitsumine.model.UserId;

import com.activeandroid.query.Delete;
import com.activeandroid.query.Select;

public class UserIdDAO {

    public static void save(String word) {
        UserId user = new UserId();
        user.word = word;
        user.save();
    }

    public static List<String> findAll() {
        List<UserId> list = new Select().from(UserId.class).execute();
        List<String> stringList = new ArrayList<>();
        for (UserId w : list) {
            stringList.add(w.word);
        }
        return stringList;
    }

    public static void delete(String word) {
        new Delete().from(UserId.class).where("word = ?", word).execute();
    }

    private UserIdDAO() {
    }
}

File: app/src/main/java/me/kirimin/mitsumine/db/UserIdDAO.kt
package me.kirimin.mitsumine.db

import com.activeandroid.Model
import java.util.ArrayList

import me.kirimin.mitsumine.model.UserId

import com.activeandroid.query.Delete
import com.activeandroid.query.Select

public class UserIdDAO private() {
    companion object {

        public fun save(word: String) {
            val user = UserId()
            user.word = word
            user.save()
        }

        public fun findAll(): List<String> {
            val list = Select().from(javaClass<UserId>()).execute<UserId>()
            val stringList = ArrayList<String>()
            for (w in list) {
                stringList.add(w.word)
            }
            return stringList
        }

        public fun delete(word: String) {
            Delete().from(javaClass<UserId>()).where("word = ?", word).execute<Model>()
        }
    }
}
